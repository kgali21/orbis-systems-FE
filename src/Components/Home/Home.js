import React from 'react';
import SymbolRequestForm from '../SymbolRequestForm/SymbolRequestForm';
import Symbols from '../SymbolDisplay/Symbols';
import SymbolRemoveForm from '../SymbolRequestForm/SymbolRemoveForm';
import styles from './Home.module.css';

const Home = () => (
    <div className={styles.App}>
    <div className={styles.Form}>
        <p>Add a stock symbol to start tracking</p>
        <SymbolRequestForm />
        <p>Remove a stock symbol</p>
        <SymbolRemoveForm />
    </div>
        <Symbols />
    </div>
)

export default Home;