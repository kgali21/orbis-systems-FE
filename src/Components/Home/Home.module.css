* {
    box-sizing: border-box;
}

.Form{
    display: flex;
    flex-direction: column;
    justify-content: space-evenly;
    align-items: center;
    width: 25vw;
    min-width: 300px;
    height: 32vh;
    background-color: #89929e;
    border-radius: 20%;
    margin: 2em 0 0 0;
}

.App{
    display: flex;
    flex-direction: column;
    align-items: center;
}

@media only screen and (max-width: 600px){
    .Form{
        display: flex;
        flex-direction: column;
        justify-content: space-evenly;
        align-items: center;
        width: 20vw;
        height: 28vh;
        background-color: #89929e;
        border-radius: 20%;
        margin: 2em 2.5em 0 0;
    }

    .App{
        display: flex;
        flex-direction: column;
        align-items: center;
    }
}