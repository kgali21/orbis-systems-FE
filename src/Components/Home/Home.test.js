import React from 'react';
import { shallow } from 'enzyme';
import Home from './Home';

describe('Header Component', () => {
    it('renders Header', () => {
        const wrapper = shallow(<Home />);
        expect(wrapper).toMatchSnapshot();
    });
});