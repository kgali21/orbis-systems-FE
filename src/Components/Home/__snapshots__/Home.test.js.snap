// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders Header 1`] = `ShallowWrapper {}`;
